adsingestp/parsers/jats.py

```diff
@@ -851,9 +852,14 @@ def _parse_pubdate(self):
             pub_format = d.get("publication-format", "")
             pub_type = d.get("pub-type", "")
             pubdate = self._get_date(d)
-            if pub_format == "print" or pub_type == "ppub" or pub_type == "cover":
+            if (
+                pub_format == "print"
+                or pub_type == "ppub"
+                or pub_type == "cover"
+                or pub_type == ""
+            ):
                 self.base_metadata["pubdate_print"] = pubdate
-            elif pub_format == "electronic" or pub_type == "epub":
+            if pub_format == "electronic" or pub_type == "epub" or pub_type == "":
```

Comment:
modified this if statement to assign the same pubdate for print and electronic if it is not specified explicitly per our slack discussions: https://adsabs.slack.com/archives/C03A3Q0H3/p1696338338852639